proposals/0000-linear-types.rst

```diff
@@ -255,7 +255,9 @@ indexed arrow.
   -> b`` (where ``a`` and ``b`` are types and ``p`` is a
   multiplicity). To avoid introducing a new notion of "mixfix"
   operators, we introduce a familly of (infix) type constructors:
-  ``(:p ->)`` for each multiplicity ``p``.
+  ``(:p ->)`` for each multiplicity ``p``. This technically steals
+  syntax as ``(:)`` is a valid type operator. But this should not be a
+  problem in practice.
```

Comment:
Is it a valid type operator?
```
Prelude> type a : b  = (a, b)

<interactive>:11:8: error:
    Malformed head of type or class declaration: a : b
```

Reply:
It seems, indeed, that you can't define a type `(:)`. All you can do is use the bad `DataKind` syntax:

```haskell
Prelude> :set -XTypeOperators
Prelude> :set -XDataKinds
Prelude> type Foo a b = a : b
```